Example/Tests/Transactions/EthTransactionBytes/EthTransactionBytesTests.swift

```diff
@@ -0,0 +1,56 @@
+//
+// Created by Timofey on 3/15/18.
+// Copyright (c) 2018 CocoaPods. All rights reserved.
+//
+
+import CryptoSwift
+import Nimble
+import Quick
+@testable import Web3Swift
+
+final class EthTransactionBytesTests: XCTestCase {
+
+    func testStaticParametersAreSignedCorrectly() {
+        expect{
+            return try EthTransactionBytes(
+                networkID: BigEndianCompactNumber(
+                    origin: BigEndianNumber(
+                        uint: 42
+                    )
+                ),
+                transactionsCount: BigEndianCompactNumber(
+                    origin: BigEndianNumber(
+                        uint: 0
+                    )
+                ),
+                gasPrice: BigEndianNumber(
+                    hex: SimpleString(
+                        string: "0x06FC23AC00"
+                    )
+                ),
+                gasEstimate: BigEndianCompactNumber(
+                    origin: BigEndianNumber(
+                        uint: 21000
+                    )
+                ),
+                senderKey: EthPrivateKey(
+                    bytes: BytesFromHexString(
+                        hex: "0x1636e10756e62baabddd4364010444205f1216bdb1644ff8f776f6e2982aa9f5"
+                    )
+                ),
+                recipientAddress: BytesFromHexString(
+                    hex: "0xcD8aC90d9cc7e4c03430d58d2f3e87Dae70b807e"
```

Comment:
@biboran Should we use `Alice().privateKey()` and `Bob().address()` in tests?

Reply:
@rockfridrich there are some problems with this approach. Some of our tests work with mutable state on the blockchain (for example `EthAutoAccountIT`). If we are going to use `Alice` and `Bob` our tests will be temporarily coupled to the greater extent that they already are. 

In this particular case I think having the data open is better than introducing any _alias_ objects because it tests low level signing maths